tools/openclaw_search.py

```
@@ -0,0 +1,23 @@
+import argparse
+from pathlib import Path
+
+def main():
+    parser = argparse.ArgumentParser(description="OpenClaw High-Speed Skill Search")
+    parser.add_argument("query", nargs="*", help="Query string")
+    parser.add_argument("--limit", type=int, default=5)
+    args = parser.parse_args()
+
+    q = " ".join(args.query).lower() if args.query else ""
+    print(f"Searching OpenClaw catalog for: '{q}'...")
+    print(f"Indexed search ready across all local skill manifests.")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)